### ISSUE TITLE
Inventory Bug | Might be related to other one.

### ISSUE THREAD

**Issue by SnakeDaoust1232** [Photon-GitHub/AACAdditionPro#53]
Info:
- Might be related to the other bug. Hope it is, and it gets fixed.

Video: https://www.youtube.com/watch?v=N5NHZ1HU5I0&feature=youtu.be

Core information

Server version: Paperspigot 1.8
AAC version: 3.1.3
AACAdditionPro Version: 1.7
ProtocolLib version: 4.2.0
Plugin list: https://gyazo.com/62230b456a22877a8803245f9132ccee

Special environment information

ViaVersion version: 1.0.4
ProtocolSupport version: 4.24
BungeeCord plugins: https://gyazo.com/b5d2b4e81fd8ee3a44b7472e9caf939d

**Comment by CeIebrimbor** [comment]
https://youtu.be/VmcBqFWZplA

It seems to be an issue with clients on a 1.8 server w/ ViaVersion. I was using a 1.11 client in the video.

I think ViaVersion messes with inventories and AAC+ thinks those packet oddities are movements/rotations.

**Comment by CeIebrimbor** [comment]
@SnakeDaoust1232 @Photon-GitHub 
I found this to ONLY occur when you have a hacked client running. They tend to manipulate inventory updating (apparently). I tested just clicking and running around with Wurst Client and got unlimited and consistent VL. When I switched to just optifine, not a single vl. So while the player doesn't have a hack running, they are still screwing with inventory updating (or something like that).

If you discover this can be reproduced on a vanilla or non-hack mod, then request reopen or post a new issue.
